### PR TITLE
Release 4.4.0

### PR DESCRIPTION
**Pull request by seut**

### CODE REVIEW

File: docs/appendices/release-notes/4.4.0.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+.. NOTE::
+
+    If upgrading a cluster, you must be running CrateDB 4.0.2 or higher before
+    you upgrade to 4.4.0.
+
+    We recommend that you upgrade to the latest 4.3 release before moving to
+    4.4.0.
+
+    A rolling upgrade from 4.2.6+ to 4.4.0 is supported.
```
Comment (mfussenegger) [inline docs/appendices/release-notes/4.4.0.rst:17]: Shouldn't this be a 4.3.x release?

**Review by mfussenegger — APPROVED** [review]
Otherwise lgtm

File: docs/appendices/release-notes/4.4.0.rst (hunk 1 above)
Comment (seut) [reply docs/appendices/release-notes/4.4.0.rst:17]: Right, thx for catching.

**Review by mechanomi — COMMENTED** [review]
lgtm

**Pull request merged**